cmd/podman/compose.go

```diff
@@ -114,15 +113,10 @@ func composeDockerHost() (string, error) {
 		return registry.DefaultAPIAddress(), nil
 	}
 
-	cfg, err := config.ReadCustomConfig()
+	// TODO need to add support for --connection and --url
```

Comment:
is this something that needs to be done before podman 5 becomes ga?

Reply:
Not really it was broken before as well: https://github.com/containers/podman/issues/20943